.dockerignore

```diff
@@ -0,0 +1 @@
+node_modules
```

Comment:
also node_modules wouldn't have to be ignored if npm install is ran with `make build`

Reply:
Building within the docker environment has proven to be a good approach in other projects I've worked in, as it means the host only needs Docker installed. Helps with Ci/CD pipelinles, etc.